### ISSUE TITLE
Support for many other things

### ISSUE THREAD

**Issue by harshilishere** [renanzdm/pos_printer_manager_forked#1]
Hey, firstly thank you so very much for the code!!!! finally something worked!!!!!
I read the code and understood that you have use webcontent to image kinda thing. Can you please tell me how to do QR codes and usual text too? 
Also consider me in I will try to actively help you out. I am very new to flutter but I will try to give my best where ever i can!

Thanks again just help  me with qr please

**Comment by harshilishere** [comment]
This worked for my in order to print full size qr codes. Hope this helps @renanzdm 


_startPrinter() async {
    //final content = Demo.getShortReceiptContent();
    String qr_data = "Hello World!";
    final content = await QrPainter(
      data: qr_data,
      version: QrVersions.auto,
      gapless: false,
    ).toImageData(100);
    //var bytes = await WebcontentConverter.contentToImage(content: content);
    //var bytes = content;
    var service = ESCPrinterService(receipt:content!.buffer.asUint8List());
    var data = await service.getBytes(paperSize: PaperSize.mm58);
    if (_manager != null) {
      print("isConnected ${_manager!.isConnected}");
      _manager!.writeBytes(data, isDisconnect: false);
    }
  }

**Comment by renanzdm** [comment]
At the moment I'm a little busy, I won't be able to think about improvements at the moment

**Comment by renanzdm** [comment]
> This worked for my in order to print full size qr codes. Hope this helps @renanzdm
> 
> _startPrinter() async {
> //final content = Demo.getShortReceiptContent();
> String qr_data = "Hello World!";
> final content = await QrPainter(
> data: qr_data,
> version: QrVersions.auto,
> gapless: false,
> ).toImageData(100);
> //var bytes = await WebcontentConverter.contentToImage(content: content);
> //var bytes = content;
> var service = ESCPrinterService(receipt:content!.buffer.asUint8List());
> var data = await service.getBytes(paperSize: PaperSize.mm58);
> if (_manager != null) {
> print("isConnected ${_manager!.isConnected}");
> _manager!.writeBytes(data, isDisconnect: false);
> }
> }

You can contribute to the package if you can, and implement standard qrCode printing

**Comment by harshilishere** [comment]
@renanzdm good idea. thank you! is there somewhere i can contribute in your package? for example app turning on and off bluetooth kinda things?

**Comment by renanzdm** [comment]
I didn't understand your question, sorry my english is not the best

**Comment by harshilishere** [comment]
If you see the rawbt app. I don't know how but on the user end when you give command to print something, the printer will turn on bluetooh, print the thing and then turn off the bluetooth. All we gotta do is to set up the printer in the settings tab for once.

Can we also do that? if so how?

**Comment by renanzdm** [comment]
this is possible to do but i don't see why, and it would also be a little work since the bluetooh connection is done through external packages besides using one for android and one for ios, so it would be too much work

**Comment by harshilishere** [comment]
can you give me some outlined plan for android? I'll give it a shot for android.
